functions/summary/app.ts

```diff
@@ -0,0 +1,41 @@
+import path from 'node:path'
+import { S3Client } from '@aws-sdk/client-s3'
+import { MetricUnits } from '@aws-lambda-powertools/metrics'
+
+import envs from '../../lib/envs'
+import { logger, metrics, middify, tracer } from '../../lib/lambda-common'
+import { getS3JSON } from '../../lib/utils'
+import { createSummary } from '../../lib/summarisation'
+import { Summary, Transcript } from '../../lib/types'
```

Comment:
Random comment... I am surprised that TS does not force you to include extensions (`.ts` or `.js`)... 🤷🏽  it must be one of the million flags you can specify in the `tsconfig.json`

Reply:
I'm not sure what dictates the behaviour here. I don't _think_ there's anything specific in the `tsconfig.json`. From [here](https://www.typescriptlang.org/tsconfig#moduleSuffixes), `.ts` seems to be inferred by default but I also have other projects where I had to include the extension in the import 🤷‍♀️